tests/mocks/slack.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from easydict import EasyDict
+
+from core.slack_client import slack_client
+
+
+@pytest.fixture()
+def slack_mock(mocker):
```

Comment:
I was trying to mock Slack calls for testing and found the VCR approach to be a bit confusing. I had problems because it was trying to compare the host, headers, and the exact URL path, which is already tested by the Slack library itself. I figured we could benefit from a simpler way to mock and would love to hear your thoughts on this.

Reply:
Maybe you can do the same thing I did on https://github.com/DjangoGirls/djangogirls/pull/833 - deleting the existing cassette files that the tests are using and regenerating them by running the tests again.

Reply:
I had a couple of issues that are worth writing down for future reference if needed:
* As to posting messages, I wanted to make sure the request to Slack was sent and went through, but Slack requires me to pass an authorization header with the API token, otherwise the response will fail, and we can't commit the token in cassette files (it was automatically being added to it).
* I couldn't test `user_invite_to_slack` because I don't have the right test environment (workspace in Enterprise Grid) as DG has to call the app with this scope, so the cassette would have failed response which I don't think makes much sense.

Overall, the gist of the issue seems to be that I'm trying to test internal stuff from DG's workspace, which I don't have access to, and I'm just not familiar with vcrpy. I guess [filtering some fields from the request](https://github.com/kevin1024/vcrpy/blob/964615af2562b24af7864456bb4c347956def424/docs/advanced.rst#filter-sensitive-data-from-the-request) should solve the issues.

Reply:
Hi @vanessa. Thank you for your work on this. We are currently on the Pro Plan and Slack gives discounts for non-profits on Pro and Business Level. Let me see if I can reach out about an upgrade so we can get on the Enterprise Grid but I have to run this by the trustees first. Will let you know once that is in order but for now I will just approve this and see what I can configure in the current set up.

Reply:
@amakarudze do we want to merge this PR and test on production?

Reply:
Yes we do @vanessa but I have been working on getting a Slack upgrade so we can be able to test it out. Right now our Pro account cannot support this change. That is the only hold-up to this. Maybe you can fix the conflicts and we merge this in while we wait?